inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/Initializer.kt

```diff
@@ -129,10 +130,23 @@ internal object Initializer {
 
             val picasso = Picasso.Builder(context)
                 .downloader(OkHttp3Downloader(client))
+                .memoryCache(LruCache(getMemoryCacheSize()))
                 .build()
             Picasso.setSingletonInstance(picasso)
         } catch (ignored: IllegalStateException) {
             // Picasso instance was already initialized
         }
     }
+
+    /**
+     * @return 1/8th of the available VM memory in byte.
+     */
+    private fun getMemoryCacheSize(): Int {
+
+        // Get max available VM memory, exceeding this amount will throw an
+        // OutOfMemory exception.
+        val maxMemory = (Runtime.getRuntime().maxMemory()).toInt()
+
+        return maxMemory / 8
```

Comment:
Is there a reason for choosing 1/8?

Reply:
According to documentation on a normal/hdpi device this is a minimum of ~ 4MB, but I still need to confirm with QA if the OOM issue is still thrown with this value.